src/apache/scala/com/datastax/sparkstress/ContextHelper.scala

```diff
@@ -1,12 +1,19 @@
 package com.datastax.sparkstress
 
 import org.apache.spark.{SparkContext, SparkConf}
+import org.apache.spark.sql.SparkSession
 
 object ConnectHelper {
 
   /*
   Get a normal SparkContext Object
    */
   def getContext(conf: SparkConf): SparkContext =
-    new SparkContext(conf)
+    this.getSparkSession(conf).sparkContext
```

Comment:
why do we need two of those classes?

Reply:
I _think_ we originally created both `apache/../ContextHelper.scala` and `dse/../ContextHelper.scala` to give us better flexibility when building against DSE vs Apache Spark.